src/templates/default_nativeapp/snowflake.yml.jinja

```diff
@@ -0,0 +1,9 @@
+native_app:
+  name: {{project_name}}
+
+  source_stage: app_src.stage
```

Comment:
We don't necessarily need this line as it's the default, but could be good for learning.

Reply:
+1, hence would like to keep this here.